doc/book/cookbook/setting-locale-without-routing-parameter.md

```diff
@@ -102,19 +102,16 @@ return [
     ]
 
     'middleware_pipeline' => [
-        'pre_routing' => [
-            [
-                'middleware' => [
-                    Application\I18n\SetLanguageMiddleware::class,
```

Comment:
To keep consistent, I would change all references to "lang" or "language" to "locale".